src/external.rs

```diff
@@ -2,6 +2,8 @@
 pub(crate) mod arbitrary_support;
 #[cfg(feature = "borsh")]
 pub(crate) mod borsh_support;
+#[cfg(feature = "nonzero")]
+pub(crate) mod nonzero_support;
```

Comment:
Since this isn't using an external library for support I think the module should live at the top-level.

Reply:
fixed